app/services/podcasts/feed.rb

```diff
@@ -9,7 +9,7 @@ def initialize(podcast)
 
     def get_episodes(limit: 100, force_update: false)
       # increased the redirect limit from 5 (default) to 7 to be able to handle such urls
-      rss = HTTParty.get(podcast.feed_url, limit: 7).body
+      rss = HTTParty.get(podcast.feed_url, limit: 7).body.to_s
```

Comment:
Minor detail here but there's a possibility of body being nil, causing the next line of code to fail. see https://app.honeybadger.io/fault/66984/605b63f08603e2ba1d9476db3b5a54c8